docs/submission/validating-molecular-data.md

```diff
@@ -0,0 +1,443 @@
+|  fileMd5sum     | files |  ![Required](/assets/submission/dictionary-required.svg)  |  Compute the md5sum of the file. This must match what is computed when the file is uploaded.  |  |  |
+|  fileType       | files |  ![Required](/assets/submission/dictionary-required.svg)  |  Data format of sequencing files.                                                             |  BAM, FASTQ          |  |
+|  fileAccess     | files |  ![Required](/assets/submission/dictionary-required.svg)  |  The permission level of a file.                                                              |  open, controlled    |  |
+|  dataType       | files |  ![Required](/assets/submission/dictionary-required.svg)  |  Descriptor of the type of file being submitted.                                              |  Submitted Reads     |  |
+### File and Data Validation Rules
+
+Sequencing data of both `BAM` and `FASTQ` type files is accepted.
+
+1. ARGO does not accept interleaved FASTQ files.
+1. Compression of FASTQ files is **required**; both _gzip_ (suffix .fq.gz or .fastq.gz) or _bz2_ (suffix .fq.bz2 or .fastq.bz2) are supported.
+1. For both FASTQ and BAM submission, all files in the `files` section must be unique in the payload.
+1. All `read_groups` in the payload and BAM header must belong to a single sample.
+1. `platform_units` must be unique with a one-to-one relationship with `submitter_read_group_id`.
+1. The total number of `read_group` objects must match the number specified in `read_group_count`.
+1. For paired end sequencing, both `file_r1` and `file_r2` are required, otherwise, only `file_r1` is required (`file_r2` must not be populated).
+1. For paired-end sequencing, `file_r1` and `file_r2` must not be the same file. 
+1. For FASTQ submission, no file can appear more than once in `file_r1` or `file_r2` across read group objects.
+
+## Validating Metadata 
+
+It is very important that molecular data is submitted with valid metadata that meets all of the rules described above.  As a helpful tool during metadata preparation, we have prepared a validation client that can be run on your data locally before submitting data officially.  The validation client will make sure that the prepared metadata and the metadata in the BAM headers is matching. 
+
+Verification will help you ensure your data is formatted correctly (with accurate identifier assignments between metadata and molecular data files) and that your submission goes smoothly while also helping the DCC ensure that the downstream [Analysis Pipelines](/docs/analysis-workflows/analysis-overview) will function seamlessly.  
+
+This validation tool will check: 
+- Data submitted in the metadata payload and BAM/FASTQ files aligns
+- Each submission is for a single sample
+- Each payload is formatted according to the rules stated above
+
+
+### Installing the Validation Client 
```

Comment:
This header can be promoted to upper level.
Basically replace `Validating Metadata` by `Installing the Validation Client`, and merge the information of two sections.

Reply:
i think it makes more sense the way it is now :) more clear with the page title

Reply:
The current organization looks good to me!